.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule ".project_extension"]
+	path = .project_extension
+	url = git@github.com:Learnosity/tools-common-extensions.git
```

Comment:
When this move, you should up a relative path https://github.com/Learnosity/api-aiproxy/blob/main/.gitmodules#L3